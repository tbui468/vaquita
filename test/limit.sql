create database sol;
open sol;

create table planets (name string, mass int, atmosphere bool);
insert into planets (name, mass, atmosphere) values ("Neptune", 20, false), ("Mars", 10, true), ("Venus", 10, false), ("Earth", 10, true);
insert into planets (name, mass, atmosphere) values ("Mercury", 5, true), ("Jupiter", 100, true), ("Saturn", 323, false);
select * from planets limit 5;
select * from planets order by id desc limit 5;
drop table planets;

close sol;
drop database sol;
exit;
