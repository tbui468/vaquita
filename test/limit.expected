created database sol
opened database sol
created table planets
inserted 4 record(s) into planets
inserted 3 record(s) into planets
1, Neptune, 20, false, 
2, Mars, 10, true, 
3, Venus, 10, false, 
4, Earth, 10, true, 
5, Mercury, 5, true, 
7, Saturn, 323, false, 
6, Jupiter, 100, true, 
5, Mercury, 5, true, 
4, Earth, 10, true, 
3, Venus, 10, false, 
dropped table planets
closed database sol
dropped database sol
